### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by marurevi**
## Mobile menu
In mobile, implement the following features:
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  menuContainer.style.textDecoration = 'none';
+  menuContainer.style.padding = '100px 0 250px 0';
+  menuContainer.style.backgroundImage = 'url(./images/image_geometry_menu_1-1.svg), url(./images/image_geometry_menu_2-1.svg)';
+  menuContainer.style.backgroundPosition = 'top, bottom';
+  menuItems.forEach((item) => { item.classList.remove('hidden'); item.addEventListener('click', menuQuit); });
+  quitButton.classList.remove('hidden');
+  quitButton.addEventListener('click', menuQuit);
+}
+
+menuButton.addEventListener('click', menuDisplay);
```
Comment (micheaol) [inline script.js:29]: Please ensure that you stick to the design as much as possible (your pop-up should have full width like the Figma design 
 and the margin between the menu item and the design) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
image  | image

File: style.css [hunk 2]
```diff
@@ -22,17 +17,34 @@ body {
...
+  .display-menu {
+    font-family: "Poppins", sans-serif;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 48px;
+    text-align: center;
+    color: #000;
+    text-decoration: none;
+  }
```
Comment (medaminedev66) [inline style.css:46]: - [ ] As the previous reviewer mentioned in the previous screenshot, the menu should cover the whole screen. I would recommend adding the `position: fixed`, `top: 0`, `right: 0`, `bottom: 0`, and `left:0` to make the menu fixed and not scrollable and cover the whole mobile screen.
- Your design:
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @marurevi 😃,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- The menu design is great and matches the Figma design. Great job 👍

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (marurevi) [reply style.css]: Indeed, the fixed position looks better. I didn't know how to fix it. Thanks

File: style.css [hunk 3]
```diff
@@ -22,17 +17,54 @@ body {
...
+    background-color: white;
+    background-position: top left, bottom left;
+    background-size: 100%, 100%;
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: 1;
+  }
```
Comment (medaminedev66) [inline style.css:66]: [OPTIONAL] The position of your menu is fixed and not scrollable, great job 👍. Let's make it better by adding `height: 100%` and removing `padding: 335px 0;` so your menu design covers the whole screen.

**Review by medaminedev66 — APPROVED** [review]
## Hi @marurevi, 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**